nicoRDL/dotTheory-JS
# JS-104 (JS-348) JavaScript Loops: Working with for Loops

## Related Ticket

https://codeandtheory.atlassian.net/browse/JS-348

## Description
- finished for loop tutorials
- completed all code challenges and quizzes

## Steps to Validate
Practice Problems
1. Open the index.html file
2. Add the consoleLogInt.js file and open the console; integers 5 to 100 should be printed
3. Add the sampleBreak.js file; the loop should break after logging 17 numbers and print "The loop has terminated..."
4. Add the log24.js file and open the console; even integers between 2 to 24 should be printed

Code Challenge
1. Open the index.html file
2. A set of 10 colored circles should be on screen
3. Refresh the page multiple times; each refresh should output a different set of 10 colors
4. Evaluate the script randomColors.js; it should be a simplified loop for producing the output above

# Risks
N/A
